Rebus.RabbitMq/RabbitMq/RabbitMqTransport.cs

```diff
@@ -170,18 +189,28 @@ void BindInputQueue(string address, IModel model)
             model.QueueBind(address, _directExchangeName, address);
         }
 
-        static void DeclareQueue(string address, IModel model, bool durable)
+        void DeclareQueue(string address, IModel model)
         {
-            var arguments = new Dictionary<string, object>
+            if (Address != null && Address.Equals(address))
             {
-                {"x-ha-policy", "all"}
```

Comment:
I am not an expert on RabbitMQ, but isn't this required for RabbitMQ clustering to work?

Reply:
Yes, I moved it to the RabbitMqQueueOptionsBuilder as a default setting (with the ability to add/or override arguments.

Reply:
⚡️ 👍